lib/statsd/instrument.rb

```diff
@@ -390,11 +390,7 @@ def legacy_singleton_client
   end
 
   def singleton_client
-    @singleton_client ||= StatsD::Instrument::Environment.from_env.client
-  end
-
-  def client
```

Comment:
Shouldn't we deprecated it first? Or a Changelog entry is enough you think ?

Reply:
I think we can get away with a CHANGELOG entry. Unless people went out of their way, they should not have been using this method.